### ISSUE TITLE
Bug: The API is loud even when asked not to be loud

### ISSUE THREAD

**Issue by riclarsson** [atmtools/arts#73]
Problem one is that Workspace outputs text despite being asked not to output text.

Code example:

```
from typhon.arts.workspace import Workspace, arts_agenda

x = Workspace(0)
```

Expected output to screen: None

Output to screen: `verbosityInit`

/////////////////////////////////////////////////////////////////////////////////////////////////

Problem two is that arts_agenda also outputs text to screen

Code:

```
from typhon.arts.workspace import Workspace, arts_agenda

def f(arts):
    arts.Print("HELLO")

x = arts_agenda(f)
```

Expected output to screen: None

Output to screen: `verbosityInit`

This bug persist even if `Workspace(0)` has been called prior to `arts_agenda(f)`

/////////////////////////////////////////////////////////////////////////////////////////////////

Problem three is something I have not found out where it is.  It has to do with `Copy` also being output to the screen.

**Comment by olemke** [comment]
I cannot reproduce problem three. If it is still present after the fix from #74 is merged, please open a new issue once you have a test case to reproduce it.

**Comment by riclarsson** [comment]
With the last fix I managed to hund down the problem.  Here is some example code to activate it:

```
import numpy as np
from typhon.arts.workspace import Workspace, arts_agenda


def geo_pos_agenda(ws):
    ws.Ignore(ws.ppath)
    ws.VectorSet(ws.geo_pos, np.array([]))


arts = Workspace(0)
arts.Copy(arts.geo_pos_agenda, arts_agenda(geo_pos_agenda))
```
